substrait_consumer/functional/common.py

```diff
@@ -1,10 +1,11 @@
+from pathlib import Path
 from typing import Callable, Iterable
 
 import pytest
 from duckdb import DuckDBPyConnection
 from ibis.expr.types.relations import Table
 
-from substrait_consumer.verification import verify_equals
+SNAPSHOT_DIR = Path(__file__).parent.parent / f"tests/functional/extension_functions"
```

Comment:
This doesn't need to be an `f-string` -- I think `flake8` should catch that.

And it probably doesn't matter, but if there's a thought that this might be run on Windows (ever) then you'll want to divide each part of the path individually so that the path separators are compatible, e.g.

```
SNAPSHOT_DIR = Path(__file__).parent.parent / "tests" / "functional" / "extension_functions"
```